### PR TITLE
Changes post ironic/inspector merge

### PR DESCRIPTION
**Pull request by elfosardo**
Remove explicit references to ironic-inspector-image as it was merged into the ironic-image container

### CODE REVIEW

**Comment by stbenjam** [comment]
Do we still want to keep this for testing older than 4.9?

**Comment by elfosardo** [comment]
> Do we still want to keep this for testing older than 4.9?

that was also my concern, I realized it right after submitting the PR
I guess I can change it and just add comments to limit using inspector-image until 4.8 included

File: common.sh [hunk 1]
```diff
@@ -160,7 +160,10 @@ export MIRROR_LOG_FILE=${REGISTRY_DIR}/${CLUSTER_NAME}-image_mirror-${OPENSHIFT_
 # Switch Container Images to upstream, Installer defaults these to the openshift version
 if [ "${UPSTREAM_IRONIC:-false}" != "false" ] ; then
     export IRONIC_LOCAL_IMAGE=${IRONIC_LOCAL_IMAGE:-"quay.io/metal3-io/ironic:master"}
-    export IRONIC_INSPECTOR_LOCAL_IMAGE=${IRONIC_INSPECTOR_LOCAL_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+# Sarting from Openshift 4.9 the ironic-inspector container is not used anymore
```
Comment (hardys) [inline common.sh:163]: Typo Starting

File: common.sh [hunk 2]
```diff
@@ -160,7 +160,10 @@ export MIRROR_LOG_FILE=${REGISTRY_DIR}/${CLUSTER_NAME}-image_mirror-${OPENSHIFT_
 # Switch Container Images to upstream, Installer defaults these to the openshift version
 if [ "${UPSTREAM_IRONIC:-false}" != "false" ] ; then
     export IRONIC_LOCAL_IMAGE=${IRONIC_LOCAL_IMAGE:-"quay.io/metal3-io/ironic:master"}
-    export IRONIC_INSPECTOR_LOCAL_IMAGE=${IRONIC_INSPECTOR_LOCAL_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+# Starting from Openshift 4.9 the ironic-inspector container is not used anymore
+    if [ "$OPENSHIFT_VERSION" <= "4.8" ]; then
```
Comment (hardys) [inline common.sh:164]: This isn't a valid test - I think you need something like `if [[ "$OPENSHIFT_VERSION" < "4.9" ]]` - that will still only do a string comparison, but that's probably good enough in this case

File: run_ci.sh [hunk 3]
```diff
@@ -104,7 +104,8 @@ if [ -d "/home/notstack/ironic-image" ] ; then
     export IRONIC_LOCAL_IMAGE=https://github.com/metal3-io/ironic-image
     export UPSTREAM_IRONIC=true
 fi
-if [ -d "/home/notstack/ironic-inspector-image" ] ; then
+# Starting from Openshift 4.9 the ironic-inspector container is not used anymore
+if [ "$OPENSHIFT_VERSION" <= "4.8" ] && [ -d "/home/notstack/ironic-inspector-image" ] ; then
```
Comment (hardys) [inline run_ci.sh:108]: Similarly this isn't a valid test, I think you'll see it generates a syntax error?

**Review by hardys — CHANGES_REQUESTED** [review]

File: common.sh (hunk 2 above)
Comment (elfosardo) [reply common.sh:164]: yeah, I know this, I don't know what I was thinking :/

File: common.sh [hunk 4]
```diff
@@ -160,7 +160,10 @@ export MIRROR_LOG_FILE=${REGISTRY_DIR}/${CLUSTER_NAME}-image_mirror-${OPENSHIFT_
 # Switch Container Images to upstream, Installer defaults these to the openshift version
 if [ "${UPSTREAM_IRONIC:-false}" != "false" ] ; then
     export IRONIC_LOCAL_IMAGE=${IRONIC_LOCAL_IMAGE:-"quay.io/metal3-io/ironic:master"}
-    export IRONIC_INSPECTOR_LOCAL_IMAGE=${IRONIC_INSPECTOR_LOCAL_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+# Starting from Openshift 4.9 the ironic-inspector container is not used anymore
+    if [[ "$OPENSHIFT_VERSION" < "4.9" ]]; then
```
Comment (dtantsur) [inline common.sh:164]: You cannot use string comparison like this.
```
$ [[ "4.10" < "4.9" ]] && echo ok
ok
```

Comment (elfosardo) [reply common.sh:164]: :facepalm: 
I swear I know this......

**Review by kirankt — APPROVED** [review]
/lgtm

**Pull request merged**